include/verilogAST.hpp

```diff
@@ -383,11 +383,13 @@ class Concat : public Expression {
 
  public:
   std::vector<std::unique_ptr<Expression>> args;
+  bool unpacked;
```

Comment:
what does this flag control? seems like it inserts a `'`? not familiar with this in verilog.

Reply:
`{x, y}` concatenates the bits of x and y together, `'{x, y}` will make it a 2-d unpacked array of bits.

Reply:
Here's an example https://verificationacademy.com/forums/systemverilog/unpacked-array-concatenation-assignment